Venus-venuslangmuir/Homework-1/99-bottles-of-beer/app.js

```diff
@@ -0,0 +1,37 @@
+/*
+  Write a script that prints the lyrics to "99 Bottles of Beer on the Wall" in the terminal. 
+  If you're unfamiliar with the song, you can [find the lyrics here](http://www.99-bottles-of-beer.net/lyrics.html).
+  
+  Be sure that all of your output includes proper spacing (no words or values running into each other).
+
+  Hint: You can make your output appear on multiple lines by using the string \n to insert a line break.
+  
+  For an extra challenge, improve this code to print "1 bottle" rather than "1 bottles." (Hint: read up on if/else
+  conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else). 
+*/
+
+function song99BottlesLyrics() { 
+  for (i = 99; i > -1; i-- ) {
```

Comment:
Where is `i` first defined?

A variable is declared the first time it appears in your script.

You can assign values to the variables when you declare them, or if your using let, you can declare them and can assign the value later in your script.